src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3735,6 +3735,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
           Outputs="$(IntermediateOutputPath)%(Culture)\$(TargetName).resources.dll"
           Condition="'@(_SatelliteAssemblyResourceInputs)' != '' and '$(GenerateSatelliteAssembliesForCore)' != 'true'">
 
+    <PropertyGroup>
+      <SdkToolsPathMaybeWithx64Architecture>$(TargetFrameworkSDKToolsDirectory)</SdkToolsPathMaybeWithx64Architecture>
```

Comment:
nit: I've seen many targets in here prepending "should have been local and private" props and items with `_`. It suggests to users to not take dependencies on that prop/item.